lib/http-revocations.js

```diff
@@ -0,0 +1,84 @@
+/*!
+ * Copyright (c) 2018-2019 Digital Bazaar, Inc. All rights reserved.
```

Comment:
it pains me to do this as I know you are super ultra ridiculously busy, but do remember the year is 2020 when you are done ensuring all functionality works, is well formatted, documented, the tests work, nothing is exploding, the goats are ok etc.

Reply:
fixed.